### ISSUE TITLE
Can't find PASSWORD_FILE

### ISSUE THREAD

**Issue by macjustice** [facebook/Recipes-for-AutoPkg#32]
I like the new password_file thingy, but including a password in the override no longer works:

```
❯ autopkg run XcodeVersionedName.munki -vv
Processing XcodeVersionedName.munki...
Use of undefined key in variable substitution: u'RECIPE_CACHE_DIR'
AppleDataGatherer
{'Input': {'appID_key': u'891bd3417a7776362562d2197f89480a8547b108fd934911bcbea0110d07f757',
           'apple_id': ### u'REDACTED',
           'password_file': u'secret.txt'}}
[Errno 2] No such file or directory: u'secret.txt'
Failed.
Receipt written to /Users/macj/Library/AutoPkg/Cache/local.munki.XcodeVersionedName/receipts/XcodeVersionedName-receipt-20191002-130802.plist

The following recipes failed:
    XcodeVersionedName.munki
        Error in local.munki.XcodeVersionedName: Processor: AppleDataGatherer: Error: [Errno 2] No such file or directory: u'secret.txt'

Nothing downloaded, packaged or imported.
```

I think it'd be fine to deprecate the override password but FYI.

**Comment by nmcspadden** [comment]
I'm honestly not sure I'd want to support allowing passing in the password via plaintext on the command line, because it's such a terrible security practice. Would you be really irritated if I removed that feature entirely?

**Comment by macjustice** [comment]
Not at all, I say toss it. Just wanted to clarify that the old way no longer works as before.

**Comment by nmcspadden** [comment]
This was fixed by your PR #35
